packages/client/src/platform.web.js

```diff
@@ -7,3 +7,7 @@ export const Response = globalThis.Response
 export const Blob = globalThis.Blob
 export const File = globalThis.File
 export const Blockstore = MemoryBlockStore
+// @ts-ignore filesFromPath any type
+export const filesFromPath = undefined
+// @ts-ignore getFilesFromPath any type
+export const getFilesFromPath = undefined
```

Comment:
Perhaps these should be more like https://github.com/ipld/js-car/blob/master/lib/indexed-reader-browser.js so that anyone trying to use them in the browser is given a reason why they're not there.

Reply:
yes, this is a good idea!